src/software/ai/evaluation/time_to_travel.cpp

```diff
@@ -0,0 +1,70 @@
+#include <cmath>
+
+#include "software/ai/evaluation/time_to_travel.h"
+
+Duration getTimeToTravelDistance(const double distance, const double max_velocity,
+                                 const double max_acceleration,
+                                 const double initial_velocity,
+                                 const double final_velocity)
+{
+    // Bound all values to be realistic
+    double d_total = std::max(0.0, distance);
+    double v_max   = std::max(0.0, max_velocity);
+    double v_i     = std::clamp(initial_velocity, -max_velocity, max_velocity);
+    double v_f     = std::clamp(final_velocity, 0.0, max_velocity);
+    double a_max   = std::max(1e-6, max_acceleration);
```

Comment:
whats the reason for using `1e-6`?

Reply:
There were two things, first being that _max_ acceleration possible by the robot should be a non-negative number, second being that we're doing lots of divisions by acceleration in the formulas, so I did not want the max acceleration to be 0 to avoid SIGFPEs.